features/cellular/framework/targets/TELIT/ME910/TELIT_ME910.h

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2017, Arm Limited and affiliates.
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef CELLULAR_TARGETS_TELIT_ME910_TELIT_ME910_H_
+#define CELLULAR_TARGETS_TELIT_ME910_TELIT_ME910_H_
+
+#ifdef TARGET_FF_ARDUINO
+#ifndef MBED_CONF_TELIT_ME910_TX
+#define MBED_CONF_TELIT_ME910_TX D1
+#endif
+#ifndef MBED_CONF_TELIT_ME910_RX
+#define MBED_CONF_TELIT_ME910_RX D0
+#endif
+#endif /* TARGET_FF_ARDUINO */
+
+#include "AT_CellularDevice.h"
+
+//the delay between sending AT commands
+#define DEFAULT_DELAY_BETWEEN_AT_COMMANDS 20
```

Comment:
Could be in `cpp` file.

Reply:
I copied this code from the implementation of the [HE910 driver](https://github.com/ARMmbed/mbed-os/blob/master/features/cellular/framework/targets/TELIT/HE910/TELIT_HE910.h#L32); should it be changed in both?

Reply:
It's enough for this PR to update this driver.

Reply:
marked as resolved but still in header?

Reply:
Oh sorry, I thought you were saying it's ok to leave this `#define` in the header for now.

Reply:
Please move this to cpp and I'll approve this pr.

Reply:
done!